app/init.php

```diff
@@ -234,15 +234,19 @@
 const METRIC_BUILDS  = 'builds';
 const METRIC_BUILDS_STORAGE  = 'builds.storage';
 const METRIC_BUILDS_COMPUTE  = 'builds.compute';
+const METRIC_BUILDS_MB_SECONDS  = 'builds.mb_seconds';
 const METRIC_FUNCTION_ID_BUILDS  = '{functionInternalId}.builds';
 const METRIC_FUNCTION_ID_BUILDS_STORAGE = '{functionInternalId}.builds.storage';
 const METRIC_FUNCTION_ID_BUILDS_COMPUTE  = '{functionInternalId}.builds.compute';
 const METRIC_FUNCTION_ID_DEPLOYMENTS  = '{resourceType}.{resourceInternalId}.deployments';
 const METRIC_FUNCTION_ID_DEPLOYMENTS_STORAGE  = '{resourceType}.{resourceInternalId}.deployments.storage';
+const METRIC_FUNCTION_ID_BUILDS_MB_SECONDS  = '{functionInternalId}.builds.mb_seconds';
```

Comment:
We might also want 
1. METRIC_FUNCTION_ID_MB_SECONDS = '{functionInternalId}.mbSeconds'

for convenience, which would be a total of the build and execution mbSeconds for a specific function

Reply:
Doesn't `METRIC_FUNCTION_ID_EXECUTIONS_MB_SECONDS` cover that?